<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DonatePanel</class>
 <widget class="QWidget" name="DonatePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>26</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DonatePanel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="2">
    <widget class="QPushButton" name="buttonWebMoney">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="QtResource.qrc">
       <normaloff>:/Resources/Donate/WebMoney.png</normaloff>:/Resources/Donate/WebMoney.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="buttonBitcoin">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="QtResource.qrc">
       <normaloff>:/Resources/Donate/Bitcoin.png</normaloff>:/Resources/Donate/Bitcoin.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonLitecoin">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="QtResource.qrc">
       <normaloff>:/Resources/Donate/Litecoin.png</normaloff>:/Resources/Donate/Litecoin.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtResource.qrc"/>
 </resources>
 <connections/>
</ui>
